Runner/JobManager.py

```diff
@@ -0,0 +1,380 @@
+        """
+        Creates the Pool that will be submitted to the batch service
+
+        :type batch_service_client: `azure.batch.BatchServiceClient`
+        :type image_references: `Utils.ImageReference`
+        """
+
+        # load the template file
+        template = ctm.load_file(self.pool_template_file)
+        
+        # set extra license if needed
+        if self.application_licenses is not None:
+            template["pool"]["applicationLicenses"] = self.application_licenses.split(",")
+
+        # Set rendering version
+        ctm.set_image_reference(template, image_references)
+        ctm.set_template_pool_id(template, self.pool_id)
+
+        all_pools = [p.id for p in batch_service_client.pool.list()]
+
+        if self.pool_id not in all_pools:
+            self.submit_pool(batch_service_client, template)
+        else:
+            Logger.info('pool [{}] already exists'.format(self.pool_id))
+
+
+    def upload_assets(self, blob_client):
+        """
+        Uploads a the file specified in the json parameters file into a storage container that will 
+        delete it's self after 7 days 
+
+        :param self: self 
+        :param blob_client: A blob service client.
+        :type blob_client: `azure.storage.blob.BlockBlobService`
+        """
+        input_container_name = "fgrp-" + self.job_id
+        output_container_name = "fgrp-" + self.job_id + '-output'
+
+        # Create input container
+        blob_client.create_container(input_container_name, fail_on_exist=False)
+        # Logger.info('creating a storage container: {}'.format(input_container_name))
+
+        # Create output container
+        blob_client.create_container(output_container_name, fail_on_exist=False)
+
+        # Logger.info('creating a storage container: {}'.format(output_container_name))
+
+        full_sas_url_input = 'https://{}.blob.core.windows.net/{}?{}'.format(
+            blob_client.account_name,
+            input_container_name,
+            Utils.get_container_sas_token(
+                blob_client,
+                input_container_name,
+                ContainerPermissions.READ +
+                ContainerPermissions.LIST))
+        full_sas_url_output = 'https://{}.blob.core.windows.net/{}?{}'.format(
+            blob_client.account_name,
+            output_container_name,
+            Utils.get_container_sas_token(
+                blob_client,
+                output_container_name,
+                ContainerPermissions.READ +
+                ContainerPermissions.LIST +
+                ContainerPermissions.WRITE))
+
+        # Set the storage info for the container.
+        self.storage_info = Utils.StorageInfo(
+            input_container_name,
+            output_container_name,
+            full_sas_url_input,
+            full_sas_url_output)
+
+        # Upload the asset file that will be rendered and
+        scenefile = ctm.get_scene_file(self.parameters_file)
+        for file in os.listdir("Assets"):
```

Comment:
📓 the path provided to the `listdir` is the base path of the file, i.e. in your case, think to remember is that the `Assets` is always going to be a set path, just mentioning for double checking.

Reply:
Thanks for checking, for now, this is how I'm handling the files. This may change in the future